mp4/emsg.go

```diff
@@ -127,5 +141,9 @@ func (b *EmsgBox) Info(w io.Writer, specificBoxLevels, indent, indentStep string
 	if b.Version == 0 {
 		bd.write(" - presentationTimeDelta: %d", b.PresentationTimeDelta)
 	}
+	level := getInfoLevel(b, specificBoxLevels)
+	if level > 0 && len(b.MessageData) > 0 {
```

Comment:
Just an idea, but maybe we should write out the length of the MessageData all the time, and include the hex values for higher levels?

Reply:
Changed as suggested.